### PR TITLE
CFE-2424:  define kept outcome with action warn if edit_line is as expected …

### PR DESCRIPTION
**Pull request by amousset**
…warn action_policy.

### CODE REVIEW

File: cf-agent/files_edit.c [hunk 1]
```diff
@@ -115,6 +115,8 @@ void FinishEditContext(EvalContext *ctx, EditContext *ec, Attributes a, const Pr
         }
         else
         {
+            cfPS(ctx, LOG_LEVEL_VERBOSE, PROMISE_RESULT_NOOP, pp, a,
+                 "No edit changes to file '%s' need saving", ec->filename);
```
Comment (ncharles) [inline cf-agent/files_edit.c]: :+1:

**Comment by amousset** [comment]
Updated commit message.

**Comment by amousset** [comment]
@jimis Rebased, Travis tests passed.

**Comment by jimis** [comment]
Thanks @amousset, can you add a "Changelog: title" tag in the commit message?

**Comment by jimis** [comment]
@nickanderson: can you have a quick look in this before I move on?

**Comment by amousset** [comment]
@nickanderson Added a second case in the test for failed outcome.

**Comment by nickanderson** [comment]
Awesome thanks! Looks good to me.

**Comment by jimis** [comment]
Merged and cherry-picked, thanks @amousset @nickanderson !

**Pull request merged**